File: ClientApp/SuspendedStorefront/src/app/api/models/customer.ts
/* tslint:disable */
/* eslint-disable */
import { ProductSubscription } from './product-subscription';
export interface Customer {
  address1?: null | string;
  address2?: null | string;
  city?: null | string;
  county?: null | string;
  id?: string;
  isActive?: boolean;
  name?: null | string;
  postalCode?: null | string;
  productSubscriptions?: null | Array<ProductSubscription>;
}

File: api/Models/Customer.cs
namespace SuspendedStorefront.Models
{
    public class Customer {
        public Guid ID { get; set; }
        public string Name { get; set; }
        public string Address1 { get; set; } = "";
        public string Address2 { get; set; } = "";
        public string City { get; set; } = "";
        public string County { get; set; } = "";
        public string PostalCode { get; set; } = "";
        public bool IsActive { get; set; }

        public IEnumerable<ProductSubscription> ProductSubscriptions { get; set; }
    }
}